packaging/rpm-ostree.spec

```diff
@@ -42,6 +42,7 @@ BuildRequires: rust
 %bcond_without zchunk
 %endif
 
+BuildRequires: libzstd-devel
```

Comment:
Let's put it together with the other "Core requirements" BRs below? And add a comment that it's a dep of the ostree-ext crate.

Reply:
done, also added a commit message with what I understand changed on ostree-ext that now requires this. But I could be wrong.